docs/docs/aleph_zero/sdk.md

```diff
@@ -429,15 +567,15 @@ invariant.off(InvariantEvent.SwapEvent, handler)
 
 ### Decimal
 
-Read more about decimals here: [Types](types.md)
+This types are utilized to represent decimal values, and it is worth noting that these decimal types are exported from Rust through WebAssembly (Wasm) using wasm-bindgen. Read more about [decimals](types.md).
```

Comment:
After reading, there is no impression that such types don't exist and what the representative examples of types are